sled-agent/src/sim/sled_agent.rs

```diff
@@ -623,6 +623,7 @@ impl SledAgent {
             chipset: Default::default(),
             devices: Default::default(),
             block_devs: Default::default(),
+            cpuid_profiles: Default::default(),
```

Comment:
`propolis_server::config::Config` has its own `Default` impl now, so we could use that instead of spelling out all of the internal fields here.